packages/react-core/src/components/Tabs/Tabs.tsx

```diff
@@ -94,6 +104,8 @@ export interface TabsProps extends Omit<React.HTMLProps<HTMLElement | HTMLDivEle
   toggleAriaLabel?: string;
   /** Callback function to toggle the expandable tabs. */
   onToggle?: (isExpanded: boolean) => void;
+  /** @beta Flag which places overflowing tabs into a menu controlled by a tab labelled "more". Additionally an object can be passed with custom settings for the overflow tab. */
```

Comment:
Maybe "triggered" or "opened" by the last tab.  The word more is configurable.
I might actually call this prop "isOverflowHorizontal".  I think that more explicitly describes the use of the flag.